### ISSUE TITLE
Add a shorthand notation for tuplets in alphaTex

### ISSUE THREAD

**Issue by cheap-glitch** [CoderLine/alphaTab#228]
# Description 
It often happens that several dozens notes in a row will be arranged in tuplets (usually triplets). Therefore it would be useful to have some kind of shorthand in order to apply tuplet logic to a whole slew of notes — similar to the way `:8` defines all the following notes as eight notes.

So instead of writing
```
:8 1.3{tu 3} 2.3{tu 3} 3.3{tu 3} 1.2{tu 3} 2.2{tu 3} 3.2{tu 3} 1.1{tu 3} 2.1{tu 3} 3.1{tu 3}
```
one could write
```
:8{tu 3} 1.3 2.3 3.3 1.2 2.2 3.2 1.1 2.1 3.1
```
which is arguably both faster to type and easier to read.

# Current Workarounds 
I guess it would be possible to write a parser to extend the alphatex before feeding it to alphatab, but it seems like reinventing the wheel.

**Comment by Danielku15** [comment]
Sounds like a good idea and should be quite easy to implement.

**Comment by Danielku15** [comment]
The shorthand notation for tuplets is now available in build 0.9.3.169. 
https://docs.alphatab.net/develop/alphatex/beat-effects#tuplet-ranges
